packages/zendesk-adapter/src/filters/dynamic_content_references.ts

```diff
@@ -50,7 +52,20 @@ const transformDynamicContentDependencies = async (
     }
     const itemInstance = placeholderToItem[placeholder[0]]
     if (!itemInstance) {
-      return [part]
+      const matches = placeholder[0].match(/.*\.(.*)\}\}/)
+      if (!matches) {
+        return [part]
+      }
+      const missingInstance = createMissingInstance(
+        ZENDESK,
+        DYNAMIC_CONTENT_ITEM_TYPE_NAME,
+        matches[1]
+      )
+      missingInstance.value.placeholder = `${placeholder[0]}`
+      return [
+        OPEN_BRACKETS,
+        new ReferenceExpression(missingInstance.elemID, missingInstance),
+        CLOSE_BRACKETS]
```

Comment:
styling nit: if you open with a newline you should also close with it
```suggestion
        CLOSE_BRACKETS,
      ]
```